src/parcsr_mv/par_csr_matop_device.c

```diff
@@ -1197,21 +1197,21 @@ hypre_ParCSRMatrixDropSmallEntriesDevice( hypre_ParCSRMatrix *A,
 
    if (type == -1)
    {
-      HYPRE_CUDA_LAUNCH( hypre_ParCSRMatrixDropSmallEntriesDevice_getElmtTols<-1>, bDim, gDim, 
```

Comment:
@waynemitchell Nice catch. Any ideas if we can catch this kind of errors in the macro `HYPRE_CUDA_LAUNCH`?

Reply:
@liruipeng I'm not sure. The gridsize and blocksize don't obey any strict rules in general do they? It seems like you would also need to pass the desired total number of threads to HYPRE_CUDA_LAUNCH() in order to check whether all the values made sense. But this would be a pretty invasive change... I don't see a clean way to catch this kind of thing.

Reply:
@waynemitchell I don't see how to check this easily either. Just felt this kind of mistake was hard to find. By `grep -R "bDim, gDim" .`, I found another one in `utilities/cuda_utils.c`, which actually is OK since both are 1 (I will make another PR to fix). Just asking if you have smart ideas :) Thanks!